draft-packages/badge/KaizenDraft/Badge/Badge.tsx

```diff
@@ -19,10 +19,32 @@ export const Badge = (props: BadgeProps) => {
       className={classNames(styles.badge, {
         [styles.default]: variant === "default",
         [styles.active]: variant === "active",
-        [styles.reversed]: reversed === true,
+        [styles.dark]: variant === "dark",
+        [styles.reversed]: reversed,
       })}
     >
       {children}
     </span>
   )
 }
+
+export const BadgeAnimated: React.FunctionComponent<BadgeProps> = props => {
+  const [isFocussed, setIsFocussed] = useState(false)
+
+  useLayoutEffect(() => {
+    setIsFocussed(true)
```

Comment:
Maybe consider using the spelling with one S (`focused`) since that's used elsewhere in Kaizen – just for cases like someone doing a search for whatever reason.

![image](https://user-images.githubusercontent.com/6406263/92195445-5ff88a00-eeb0-11ea-898e-e8cad51e22b2.png)
![image](https://user-images.githubusercontent.com/6406263/92195468-6424a780-eeb0-11ea-85ea-fc898228339d.png)

Reply:
Ahahaha, yes oh I have spent hours bike shedding this at a past job, I shall update